docs/source/app_developers_guide/aws.rst

```diff
@@ -38,349 +41,391 @@ with a validator, a REST API, and three transaction processors.
 This demo environment introduces basic Sawtooth functionality with the
 IntegerKey and Settings transaction processors for the business logic
 and the Sawtooth CLI as the client. It also includes the XO transaction
-processor, which is used in a later tutorial.
+processor, which is used in the advanced tutorials.
+
+The Amazon Machine Image (AMI) for Sawtooth has a ``systemd`` service that
+handles environment setup steps such as generating keys and creating a
+genesis block. To learn how the typical startup process works, see
+:doc:`ubuntu`.
+>>>>>>> Rewrite AWS tutorial for setting up app dev environment
 
 
 Launching a Sawtooth Instance
 =============================
 
-Launch your instance from the
-`Sawtooth product page <https://aws.amazon.com/marketplace/pp/B075TKQCC2>`_
-on the AWS Marketplace. Detailed instructions for
-Launching an AWS Marketplace Instance can
-be found `here <http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/launch-marketplace-console.html>`_.
+#. Launch a Sawtooth instance from the `Hyperledger Sawtooth product page
+   on the AWS Marketplace <https://aws.amazon.com/marketplace/pp/B075TKQCC2>`_.
 
-.. note::
+   For more information, see the Amazon guide
+   `Launching an AWS Marketplace Instance
+   <http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/launch-marketplace-console.html>`_.
 
-  The default security group with recommended settings for Sawtooth is
-  configured to only allow ssh traffic inbound.
+   .. note::
 
-  * If you'd like to attach a transaction processor remotely, you'll need
-    to add an inbound rule allowing TCP traffic on port 4004.
+      The default security group with recommended settings for Sawtooth
+      allows inbound SSH traffic only.
 
-  * If you'd like to access the REST API remotely, you'll need to add an
-    inbound rule allowing TCP traffic on port 8008.
+      * To attach a transaction processor remotely, add an inbound rule to
+        allow TCP traffic on port 4004.
 
-  Please see Amazon's `Security Groups
-  <http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_SecurityGroups.html#AddRemoveRules>`_
-  documentation for assistance with editing security group rules.
+      * To access the REST API remotely, add an inbound rule to allow TCP
+        traffic on port 8008.
```

Comment:
Maybe also add a note about port 8800 for adding validators?

Reply:
Good idea - thanks!  I just added a commit with this change.